examples/marketing/banner.rb

```diff
@@ -0,0 +1,44 @@
+require 'bigcommerce'
+
+Bigcommerce.configure do |config|
+  config.auth = 'legacy'
+  # You will get this url when registering for an API key
+  config.url = ENV.fetch['BC_STORE_HASH']
+  config.username = ENV.fetch['BC_CLIENT_ID']
+  config.api_key = ENV.fetch['BC_ACCESS_TOKEN']
+end
+
+puts Bigcommerce::System.time
+puts Bigcommerce.api_limit
+
+# List banners
+@banner = Bigcommerce::Banner.all
+puts @banner
+
+# Get a banner
+puts Bigcommerce::Banner.find(@banner[0].id)
+
+# Get a count of banners
+puts Bigcommerce::Banner.count
+
+# Create a banner
+@banner = Bigcommerce::Banner.create(
+	name: "Fourth of July Banner",
+	content: "We're having a 10% sale for our grand closing until February 24th! Use code: DUSTY10 at Checkout",
+	page: "home_page",
+	location: "top",
+	date_type: "always"
+)
+puts @banner
+
+# Update a banner
+puts Bigcommerce::Banner.update(
+	@banner.id,
+	content: "I have been updated."
+)
```

Comment:
Would prefer to call `@banner.update(content: "wow such update")` than `Bigcommerce::Banner.update(@banner.id, content: "wow such update")`